setup.py

```diff
@@ -142,7 +142,6 @@ def readfile(fname):
         'Hierarchical Dirichlet Process, HDP, Random Projections, '
         'TFIDF, word2vec',
 
-    license='LGPL',
```

Comment:
Our license is standard so has a Trove Classifier specified later.

 From the [setup.py](https://docs.python.org/2/distutils/setupscript.html) guide
> The license field is a text indicating the license covering the package where the license is not a selection from the “License” Trove classifiers. See the Classifier field. Notice that there’s a licence distribution option which is deprecated but still acts as an alias for license.